include/crpropa/Module.h

```diff
@@ -59,7 +59,17 @@ class AbstractCondition: public Module {
 	void setMakeAcceptedInactive(bool makeInactive);
 	void setRejectFlag(std::string key, std::string value);
 	void setAcceptFlag(std::string key, std::string value);
+
+	void printRejectFlag();
```

Comment:
Do we need the print methods or are they just for debugging?

Reply:
This is intended for the debugging. Up to now there is no getter function for the key and value of the reject/accept flags. 
One could also think about a real get-function, but this would return a pair of strings, where users can easily be confused by the ordering of the key / value.

Reply:
I have changed it to a get function which returns key & value.